### PR TITLE
Drone shell changes

### LINKED ISSUE #24376: Derelict drone shell spawn
Problem Description: Unactivated drone shell is blocking the way

What did you expect to happen: I wanted to get access to SMES

What happened instead: Unactivated drone shell is blocking the way to SMES

Why is this bad/What are the consequences: You can't move this shell, so you can't access the SMES unless you're spawned in that shell/someone else spawns in it/you dismantle the reinforced wall to get access to SMES

Steps to reproduce the problem: Spawn as a derelict drone that not the problem one, go to the Aft Solars.

Server: Bagil

Revision: Server revision compiled on: 2017-02-23
derel

### PR DESCRIPTION
**Pull request by coiax**
Fixes #24376.

- Drone shells are now points of interest.
- Derelict drone shells are now just dusty drone shells, rather than mob spawning effects.

:cl: coiax
add: Drone shells are now points of interest in the orbit list.
add: Derelict drone shells now spawn with appropriate headgear.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -182,3 +182,7 @@
 
 /mob/living/simple_animal/drone/cogscarab/ratvar_act()
 	fully_heal(TRUE)
+
+/obj/item/drone_shell/dusty
+	name = "dust-caked drone shell"
```
Comment (GunHog) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: Please rename this. I keep confusing it with "dust-caked golem shell". I am excited about golems, but not drones.

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: rude

Comment (GunHog) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: Correction: I am **more** excited about golems than I am drones. It is just that I do not like getting them mixed up!

**Comment by optimumtact** [comment]
You're the steelpoint of drones

**Pull request merged**